common/config/rush/browser-approved-packages.json

```diff
@@ -2,6 +2,10 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/approved-packages.schema.json",
   "packages": [
+    {
+      "name": "doc-plugin-jekyll",
+      "allowedCategories": [ "libraries" ]
+    },
     {
```

Comment:
Move this to nonbrowser-approved-packages.json

Reply:
(To get Rush to sort+autoformat the files for you: after you move the JSON entry, delete `"libraries"` from your  array, then run `"rush update"`.  I've been wanting to improve this experience heheh...)